generators/app/templates/src/constants/routes.ejs

```diff
@@ -0,0 +1,9 @@
+exports.InitialLoading = 'InitialLoading';
+<%_ if(features.login) { _%>
+exports.Login = 'Login';
+<%_ } _%>
+exports.Home = 'Home';
+<%_ if(features.tabs) { _%>
+exports.Tab1 = 'Tab1';
+exports.Tab2 = 'Tab2';
```

Comment:
Little annoying comment: if `Tab1` and `Tab2` depends on `Home` screen, I'd rename these literals to `'Home/Tab1'` and `'Home/Tab2'` as well.
I don't know if this applies here, but it would be great to use this as a convention 😇

Reply:
agreed